### PR TITLE
Cherry pick commits and prepare version 0.2.2

### PR DESCRIPTION
**Pull request by jeremy-davis-sonarsource**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -68,10 +68,49 @@ To build the lib, run:
 yarn build
 ```
 
+### Generated files
+
+The lib is built on top of some generated files like the design-tokens of the icons fonts. Those generated files are versioned in the `src/generated` and `i18n` folders.
```
Comment (david-cho-lerat-sonarsource) [inline README.md:73]: `like the design-tokens of the icons fonts`: was this `of` meant to be an `or`?

File: README.md [hunk 2]
```diff
@@ -68,10 +68,49 @@ To build the lib, run:
...
+
+```bash
+yarn build-fonts
+```
+
+#### Generated Design Tokens
+
+Our raw design tokens files that are created on Figma are stored inside the `design-tokens/tokens` folder. We can't use these tokens as is, so we transform them using `style-dictionary` to css variable format and store them in the `src/generated` folder. We also generate a few other files to help us with typings.
+
+If new design tokens are added to the raw figma file, we must run the following command to generate the new design tokens css variables:
```
Comment (david-cho-lerat-sonarsource) [inline README.md:89]: NP: `Figma`

**Review by david-cho-lerat-sonarsource — APPROVED** [review]
LGTM - just a question about an `of` that maybe should be an `or` in the README :+1:

File: README.md (hunk 1 above)
Comment (jeremy-davis-sonarsource) [reply README.md:73]: I'm not sure it's worth fixing here, but definitely on main.
I made a note of it and will create a PR with these

File: README.md (hunk 2 above)
Comment (jeremy-davis-sonarsource) [reply README.md:89]: Same as above

**Pull request merged**